James issue#52

j2quik opened the pull request:
Forgot to add the test cases with the original pull request associated with Issue#27.

j2quik committed:
Added test cases for sun.py

Forgot to add the test cases with the original pull request associated with Issue#27.

JosephBehr reviewed (changes requested):
Good tests added to the code, did you make sure they pass with pytest? Also, remeber to include  reference to issues, @returns, @param[in], and @param[out] in front of relevant doxygen comments for literate programming and automatic documentation.

j2quik committed:
Updated test case for sun.py

This file only has test cases for sun.py. Added reference to Isssue#51 and relevant Doxygen comments for literate programming and automatic documentation.

j2quik commented:
I did run the test and it passed. I only included the test for sun.py in the updated file. Once approved, I will merge into test.py. I made the suggested improvements.

j2quik committed:
Updated test.py

Integrated test cases for sun.py into test.py

JosephBehr reviewed (approved):
Awesome, thanks for making the changes
